moveit2.repos

```diff
@@ -35,3 +35,9 @@ repositories:
     type: git
     url: https://github.com/JafarAbdi/fake_joint
     version: foxy
+
+# TODO(YuYan): Switch to upstream once PR https://github.com/ros-planning/warehouse_ros/pull/48 is merged
+  warehouse_ros:
```

Comment:
You could add the "warehouse_ros_mongo" repository as well

Reply:
Done. Thanks for pointing this out!